docs/historic/NEWS.rst

```diff
@@ -2248,7 +2248,7 @@ Features
    st_dev, st_nlink, st_uid, and st_gid fields.  This is in
    preparation for the deprecation of FilePath.statinfo. (#4712)
  - IPv4Address and UNIXAddress now have a __hash__ method. (#4783)
- - twisted.protocols.ftp.FTP.ftp_STOR now catches `FTPCmdError`s
+ - twisted.protocols.ftp.FTP.ftp_STOR now catches ``FTPCmdError`` s
```

Comment:
erm, do we have to write news files in ReST now?

Reply:
considering the file format is ".rst", I am guessing that writing them in rest compatible syntax is a good idea, yeah